Source/Private/_ASHierarchyChangeSet.mm

```diff
@@ -300,6 +310,9 @@ - (void)moveItemAtIndexPath:(NSIndexPath *)indexPath toIndexPath:(NSIndexPath *)
   /**
    * TODO: Proper move implementation.
    */
+  [self _ensureNotCompleted];
+  _ASHierarchyMoveItemChange *change = [[_ASHierarchyMoveItemChange alloc] initWithChangeType:_ASHierarchyChangeTypeMove fromIndexPath:indexPath toIndexPath:newIndexPath animationOptions:options];
+  [_moveItemChanges addObject:change];
   [self deleteItems:@[ indexPath ] animationOptions:options];
   [self insertItems:@[ newIndexPath ] animationOptions:options];
```

Comment:
The primary reason of keeping these (for now) is that `move` nodes shall be split and processed in `delete` and `insert` during the batch process. It just skips calling the UICollectionView delete/insert, but move instead.

We can consider the implementation like reload. Yet, I'm not sure where is the appropriate place to process the split.